### ISSUE TITLE
Helm chart kubernetes

### ISSUE THREAD

**Issue by wellbastos** [Swatto/promtotwilio#4]
I created a helm chart of kubernetes for this project, to use  with prometheus server.
Send  permission to me,  to create a pr with these modifications.

**Comment by Swatto** [comment]
Hi. Thanks for this proposal. Feel free to create a PR and I’ll have a look.

**Comment by wellbastos** [comment]
Hi,  I received error 403, dont has  permission.
remote: Permission to Swatto/promtotwilio.git denied to wellbastos.
fatal: unable to access 'https://github.com/Swatto/promtotwilio.git/': The requested URL returned error: 403

**Comment by Swatto** [comment]
True: this is a repo on my account but we don’t have to give permissions to accept a PR (it would be insane for big open source project).

To solve this, you have to fork it to create your own repo of mine. Then push the branch to your own repo and then create the PR. You can find more instructions [here](https://kbroman.org/github_tutorial/pages/fork.html)
